### DISCUSSION TITLE
Can someone tell me is there anyway too reach other Git Hub users on GitHub?

### DISCUSSION THREAD

**Discussion by DataJuggler** [community/community#125260] [Repositories]
### Select Topic Area

Question

### Body

My NuGet package DataJuggler.Blazor.Components, is approaching 200,000 installs, so I spend a month, create a new Calendar Component and completely redo my Combo Box

![Calendar](https://github.com/community/community/assets/26331133/f282b1b0-5c73-40e2-bde7-b0e0c2a391b7)

And created a new video, and YouTube decides to show it to less than 40 people after 3 days.

Is there any place on this site to reach the C# / Blazor users that might be interested in a free set of Blazor Controls, a sample project to see how to use it, and a video that comes with it?

I feel like out of all the C# Blazor users, at least 2 per 100 will find this project worth the price (free).

DataJuggler.Blazor.Components
https://github.com/DataJuggler/DataJuggler.Blazor.Components

Sample Project

NTouch
https://github.com/DataJuggler/NTouch

Video

First Opensource Saturday - Sunday Edition
https://youtu.be/uxa1xR6xpzk

Thank you

**Comment by queenofcorgis** [comment]
Hi @DataJuggler, 

Thanks for sharing your project with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members. I'd suggest looking for communities with repos and Discussions dedicated to C# and Blazor users. 

Good luck!
